.github/workflows/update_docs.sh

```diff
@@ -1,43 +1,40 @@
-#!/bin/sh
+#!/usr/bin/env bash
 set -e
 
-echo "\n-o- Checkout fresh branch -o-"
-git checkout -b update_changelog
-
-echo "\n-o- Setting commit user -o-"
+echo -e "\n-o- Setting commit user -o-"
 git config --local user.email "dev@optimade.org"
 git config --local user.name "OPTIMADE Developers"
 
-echo "\n-o- Install current SHA (optimade) -o-"
-pip install -U -e .[server]
-pip install invoke jsondiff
-
-echo "\n-o- Update 'API Reference' docs -o-"
+echo -e "\n-o- Update 'API Reference' docs -o-"
 invoke create-api-reference-docs --pre-clean
 
-echo "\n-o- Commit update - API Reference -o-"
+echo -e "\n-o- Commit update - API Reference -o-"
 git add docs/api_reference
 if [ -n "$(git status --porcelain --untracked-files=no --ignored=no)" ]; then
     # Only commit if there's something to commit (git will return non-zero otherwise)
     git commit -m "Release ${GITHUB_REF#refs/tags/} - API Reference"
 fi
 
-echo "\n-o- Update version -o-"
-invoke setver -v ${GITHUB_REF#refs/tags/}
+echo -e "\n-o- Update version -o-"
+invoke setver --ver="${GITHUB_REF#refs/tags/}"
 
-echo "\n-o- Generate changelog -o-"
-apt-get update
-apt-get -y install ruby-full
+echo -e "\n-o- Generate changelog -o-"
+sudo apt-get update && sudo apt-get -y install ruby-full
 export GEM_HOME="${HOME}/gems"
 export PATH="${GEM_HOME}/bin:${PATH}"
 gem install github_changelog_generator
 github_changelog_generator --user "Materials-Consortia" --project "optimade-python-tools" --token ${INPUT_TOKEN}
```

Comment:
Looks like this has been bundled into an action, which might simplify things slightly?

https://github.com/marketplace/actions/generate-changelog

Reply:
Right! This was originally used as well by @shyamd I believe, but I had to move to another solution due to the old version of `push-protected`. This is not necessary anymore - we should use the action again.
I'll make another commit for this.